joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/IdentifierTests.scala

```diff
@@ -54,16 +54,24 @@ class IdentifierTests extends RubyCode2CpgFixture {
         |  end
         |end
         |
+        |Person p
```

Comment:
`Person p` doesn't mean what you probably intended it to (and isn't meaningful Ruby): here, it's being parsed as a command (i.e. the invocation of a method named `Person` with argument `p`.)

Reply:
Fixed this.